lib/servers/Pulser2/pulse_sequences/pulse_sequence.py

```diff
@@ -12,7 +12,7 @@ class pulse_sequence(object):
 	required_subsequences = []
 	replaced_parameters = {}
 	
-	def __init__(self, parameter_dict, start = WithUnit(0, 's')):
+	def __init__(self, parameter_dict, start = WithUnit(1.0, 'us')):
```

Comment:
@theoriginaljuice @jayich Ok, the problem here is that the pulser hardware will not take a t=0 time for the initial DDS pulse. There is a note on this at line 107 in DDS.py. The way the haeffner group handles this is by adding self.end = WithUnit(10, 'us') at the beginning of all pulse sequences. Im not sure what the deal is here but i dont think all pulse sequences should start with self.end = ''. any suggestions would be helpful